sycl/test-e2e/Basic/max_work_group_query.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
+// REQUIRES: cuda || hip || level_zero
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
Is GPU the only supported device?
Usually we test on 3 types of devices:
```
// RUN: %CPU_RUN_PLACEHOLDER %t.out
// RUN: %GPU_RUN_PLACEHOLDER %t.out
// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Reply:
I was bit hesitant to check for CPU, ACC as they could just yield the pre-set limits of `<int>::max`

Reply:
> I was bit hesitant to check for CPU, ACC as they could just yield the pre-set limits of `<int>::max`

I think they will be skipped. AFAIK, tested backends do not support CPU or ACCELERATOR types of devices. As we skip OpenCL, there is no point to test on CPU and ACCELERATOR devices.